src/org/ods/orchestration/util/Project.groovy

```diff
@@ -1980,4 +2018,26 @@ class Project {
         }
     }
 
+    void addDefaults(String component) {
+        def rmURL = git.getOriginUrl()
```

Comment:
For me, rmURL is not abvious at frist sight. I think we can write releaseManagerURL.